### DISCUSSION TITLE
Migrate docker/postgres to docker/sqlite

### DISCUSSION THREAD

**Discussion by stueja** [jonaswinkler/paperless-ng#1446] [Support]
Is it possible to migrate a paperless docker with postgres to a paperless docker with sqlite? If so, is that recommended, or rather not? Currently, my paperless holds 1500 documents.

**Comment by TheFehr** [comment]
There is always a way. But may I ask what is your reason to do it?

**Reply by stueja** [reply]
Sure! Backupability in my case. Currently, it seems easier to me to backup and eventually restore one sqlite file than the volume of a docker container holding the folders with the files for a postgres database.

**Reply by TheFehr** [reply]
Okay.
There is no paperless-ng specific solution for such a migration.
But I'd asume if you manage to migrate the data and just hock the new sqlite file up to paperless-ng it should work.

Probably worth booting up 2 new fresh throwaway instances, one with sqlite and one with postgres.
So you have clean db's in both. And such can compare where you would need to migrate what to.
There are some proprietary softwares for windows that offer gui's to migrate between different database backends.

**Reply by stueja** [reply]
What about export from the paperless-ng docker/postgres installation and import it into a fresh install of paperless-ng on docker/sqlite? Would that keep tags, correspondents, meta information?

**Reply by TheFehr** [reply]
As far as I know it only exports the documents. No additional metadata.
One option would be to use the api to copy all the metadata.

**Comment by stueja** [comment]
Trying a couple things out I believe I will stick with postgres in docker and simply run the document_exporter on a regular basis as a backup and re-import the exported files and manifest.json when needed.
Thanks for your ideas!
